src/extension.ts

```diff
@@ -144,13 +144,14 @@ export async function activate(context: vscode.ExtensionContext) {
             Commands.register('aws.listCommands', () =>
                 vscode.commands.executeCommand('workbench.action.quickOpen', `> ${getIdeProperties().company}:`)
             ),
-            Commands.register('aws.credential.profile.create', async () => {
+            Commands.register('aws.credentials.profile.create', async () => {
                 try {
                     await globals.awsContextCommands.onCommandCreateCredentialsProfile()
                 } finally {
                     telemetry.recordAwsCreateCredentials()
                 }
             }),
+            Commands.register('aws.credentials.edit', () => globals.awsContextCommands.onCommandEditCredentials()),
```

Comment:
Is it worth it to try and get telemetry on this to understand how many people are editing credentials?

Reply:
Not really due to https://github.com/aws/aws-toolkit-vscode/pull/2650.
https://github.com/aws/aws-toolkit-vscode/pull/2745 will make this instrumentation (almost) free though.